content/en/database_monitoring/setup_mysql/azure.md

```diff
@@ -0,0 +1,321 @@
+init_config:
+instances:
+  - dbm: true
+    host: <INSTANCE_ADDRESS>
+    port: 3306
+    username: datadog
+    password: <UNIQUEPASSWORD" \
+  datadog/datadog
+```
+
+### Configure with mounted files
+
+To configure a cluster check with a mounted configuration file, mount the configuration file in the Cluster Agent container on the path `/conf.d/mysql.yaml`:
+
+```yaml
+cluster_check: true  # Make sure to include this flag
+init_config:
+instances:
+  - dbm: true
+    host: '<AZURE_INSTANCE_ENDPOINT>'
+    port: 3306
+    username: datadog
+    password: '<UNIQUEPASSWORD>'
+```
+
+### Configure with Kubernetes service annotations
+
+Rather than mounting a file, you can declare the instance configuration as a Kubernetes Service. To configure this check for an Agent running on Kubernetes, create a Service in the same namespace as the Datadog Cluster Agent:
+
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+  labels:
+    tags.datadoghq.com/env: '<ENV>'
+    tags.datadoghq.com/service: '<SERVICE>'
+  annotations:
+    ad.datadoghq.com/service.check_names: '["mysql"]'
+    ad.datadoghq.com/service.init_configs: '[{}]'
+    ad.datadoghq.com/service.instances: |
+      [
+        {
+          "dbm": true,
+          "host": "<AZURE_INSTANCE_ENDPOINT>",
+          "port": 3306,
+          "username": "datadog",
+          "password": "<UNIQUEPASSWORD>"
+        }
+      ]
+spec:
+  ports:
+  - port: 3306
+    protocol: TCP
+    targetPort: 3306
+    name: mysql
+```
+
+The Cluster Agent automatically registers this configuration and begin running the MySQL check.
+
+To avoid exposing the `datadog` user's password in plain text, use the Agent's [secret management package][4] and declare the password using the `ENC[]` syntax.
+
+[1]: /agent/cluster_agent
+[2]: /agent/cluster_agent/clusterchecks/
+[3]: https://helm.sh
+[4]: /agent/guide/secrets-management
+{{% /tab %}}
+{{< /tabs >}}
+
+### Validate
+
+[Run the Agent's status subcommand][6] and look for `mysql` under the Checks section. Or visit the [Databases][7] page to get started!
+
+## Install the Azure MySQL Integration
+
+To collect more comprehensive database metrics from Azure, install the [MySQL integration][8] (optional).
+
+## Troubleshooting
+
+If you have installed and configured the integrations and Agent as described, and it is not working as expected, see [Troubleshooting][9].
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /agent/basic_agent_usage#agent-overhead
+[2]: /database_monitoring/data_collected/#sensitive-information
+[3]: https://docs.microsoft.com/en-us/azure/mysql/howto-server-parameters
+[4]: https://dev.mysql.com/doc/refman/8.0/en/creating-accounts.html
+[5]: https://app.datadoghq.com/account/settings#agent
+[6]: /agent/guide/agent-commands/#agent-status-and-information
+[7]: https://app.datadoghq.com/databases
+[8]: /integrations/azure_db_for_mysql
+[9]: /database_monitoring/troubleshooting/?tab=mysql
```

Comment:
```suggestion
[9]: /database_monitoring//setup_mysql/troubleshooting
```

Reply:
looks like you added an extra `/` here, ill fix after